<?xml version="1.0"?>
<launch>
    <node   pkg="tello_controller"  type="show_way_point" name="way_point" output="screen">
        <param name="prefix"            type="string"   value="depth_cam_" />
        <param name="wpFrame"           type="string"   value="camera7_color_optical_frame"/>
        <param name="way_point_file"    type="string"   value="$(find tello_controller)/way_point/mini_sample_pnt.txt" />
        <param name="all_pub"           type="bool"     value="false"/>
    </node>
    <node   pkg="tello_controller"  type="show_way_point" name="pozyx" output="screen">
        <param name="prefix"            type="string"   value="pozyx_" />
        <param name="wpFrame"           type="string"   value="camera7_color_optical_frame"/>
        <param name="way_point_file"    type="string"   value="$(find tello_controller)/way_point/pozyx.txt" />
        <param name="all_pub"           type="bool"     value="false"/>
    </node>
</launch>
